igor-web/src/main/groovy/com/netflix/spinnaker/igor/docker/DockerMonitor.groovy

```diff
@@ -177,7 +181,11 @@ class DockerMonitor extends CommonPollingMonitor<ImageDelta, DockerPollingDelta>
 
     @Override
     protected Integer getPartitionUpperThreshold(String partition) {
-        return dockerRegistryAccounts.accounts.find { it.name == partition }?.itemUpperThreshold
+        def upperThreshold = dockerRegistryAccounts.accounts.find { it.name == partition }?.itemUpperThreshold
```

Comment:
As far as I can tell, this property will never exist?
https://github.com/spinnaker/clouddriver/blob/master/clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/security/DockerRegistryNamedAccountCredentials.groovy